docs/userguide/execution.rst

```diff
@@ -39,7 +39,7 @@ parameters include access keys, instance type, and spot bid price
 Parsl currently supports the following providers:
 
 1. `parsl.providers.LocalProvider`: The provider allows you to run locally on your laptop or workstation.
-2. `parsl.providers.CobaltProvider`: This provider allows you to schedule resources via the Cobalt scheduler. **This provider is deprecated and will be removed by 2024.04**.
+2. `parsl.providers.CobaltProvider`: This provider allows you to schedule resources via the Cobalt scheduler. This Provider is DEPRECATED, and will be removed by 2024.04.
```

Comment:
I would point out two issues here:
1. This change does not sound related to the PR title. It is good practice to keep PRs limited in scope.
2. The existing text is reasonable enough, and the new text has some odd capitalization of `provider`
So, I'd recommend removing these changes from this PR.